packages/js/product-editor/src/components/variations-table/variation-actions-menus/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const QUICK_UPDATE = 'quick-update';
```

Comment:
I think this should be changed to `export const MULTIPLE_UPDATE = 'multiple-update';`